<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ondas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background:rgb(68, 69, 84);
	color:rgb(23, 33, 33);
}

QLabel {
	background:rgb(191, 172, 181);
	border-radius:5px;
	
	

}
QPushButton{
	background:rgb(127, 123, 130);
	color:rgb(229, 208, 204);
	font-size:8pt;
	padding: 10px;
	border-radius:2px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>651</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_ask">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>ASK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_fsk">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>FSK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_psk">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PSK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cuatro_psk">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>4-PSK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ochoqam">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>8-QAM</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>330</y>
      <width>651</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_unipolar">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Unipolar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_nrzl">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NRZL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_nrzi">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NRZ-I</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_rz">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>RZ</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btn_manchester">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Manchester</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_manchester_diff">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Manchester differencial</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_ami">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>AMI</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_b">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>B82S</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="btn_hd">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text">
          <string>HDB3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>649</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Modulaci贸n digital/digital</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>649</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Modulaci贸n por desplazamiento</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>600</y>
      <width>651</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="btn_analogico">
       <property name="font">
        <font>
         <family>Yu Gothic UI</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>510</y>
      <width>651</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Modulaci贸n de anal贸gico a digital</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
